src/Trace/Tracer.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace OpenTelemetry\Trace;
+
+class Tracer extends \OpenTelemetry\Tracing\Tracer
```

Comment:
I think you were renaming Tracing to Trace.  Did you mean to change it here too?

Reply:
I followed instructions from here: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/library-layout.md and I want to rename Tracing to Trace.
For now, I just extended the previous implementation to make sure that it's ok for you to rename. After we decide this is the right path, I will move the implementation here.